src/main/java/com/mk/cti/controller/PhoneController.java

```diff
@@ -30,6 +34,18 @@
 
         PhoneResponse phoneResponse = phoneResponseService.findByPhoneNumber(phoneNumber);
 
+        if(phoneResponse == null) {
+            phoneResponse = new PhoneResponse();
+            phoneResponse.setCustomerId(0L);
+            phoneResponse.setLoanId(0L);
+            phoneResponse.setAuthCode("A093756186");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "A093756186" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1192)